Add PingCAP use case (#68)
grpc/grpc.io: content/about.md

         Last but not least, while there are a number of network communicationprotocols for configuration, operational state retrieval and network telemetry, gRPC provides us with a unified flexible protocol and transport to ease client/server interaction.
    </div>

</div>

<div class="testimonialsection">
    <div class="testimonialimage"><a href="http://www.pingcap.com">
        <img src="../img/pingcap5.png" /></a></div>
        <div>
            <div class="testimonialquote">“ </div>

>             At PingCAP, we use gRPC in almost every project that involves interactions or communications through network. As has been proven, gRPC has well met our demands in terms of performance and usability. We also provide gRPC library for Rust, which is the first production-ready gRPC implementation in the Rust community.

Author: ```suggestion
            At PingCAP, we use gRPC in almost every project that involves interactions or communications through network. As has been proven, gRPC has well met our demands in terms of performance and usability. We also provide a production-ready gRPC library for Rust.
```

>         </div>

Reviewer: The claim "first" is hard to prove. grpc.io doesn't want to endorse such claims. Can you please reword it to not include "first"

Author: Thanks for the comment. What about "which is among the first few production-ready gRPC implementations in the Rust community"

Reviewer 2: Thanks for the suggestion @dcalvin. I'll let @thisisnotapril comment on it.

How about simply:
> We also provide a production-ready gRPC library for Rust.

@thisisnotapril WDYT?

Author: @chalin Your suggestion sounds good to me! We are good with either your simple version or mine.  I will update when @thisisnotapril comments.

Author: @chalin Updated as you suggested.
